task/prepare.go

```diff
@@ -52,7 +52,7 @@ var allProfiles = []livepeerAPI.Profile{
 	},
 }
 
-func Prepare(tctx *TaskContext, assetSpec *livepeerAPI.AssetSpec, file io.ReadSeekCloser) (string, error) {
+func Prepare(tctx *TaskContext, assetSpec *livepeerAPI.AssetSpec, file io.ReadSeekCloser, size int64, reportProgressStartPercentage int) (string, error) {
```

Comment:
**nit:** `assetSpec` already has the `size` afaik, so we could skip the extra `size` var

Reply:
Also, an idea to avoid the long var name
```suggestion
func Prepare(tctx *TaskContext, assetSpec *livepeerAPI.AssetSpec, file io.ReadSeekCloser, size int64, progressStart int) (string, error) {
```